autoload/rainbow_csv.vim

```diff
@@ -821,12 +821,25 @@ func! rainbow_csv#align_field(field, is_first_line, max_field_components_lens, i
 endfunc
 
 
-func! rainbow_csv#csv_align()
+func! rainbow_csv#csv_align() range
+    let first_line = a:firstline
+    let last_line = a:lastline
     " The first (statistic) pass of the function takes about 40% of runtime, the second (actual align) pass around 60% of runtime.
     " Numeric-aware logic by itself adds about 50% runtime compared to the basic string-based field width alignment
     " If there are lot of numeric columns this can additionally increase runtime by another 50% or more.
     let show_progress_bar = wordcount()['bytes'] > 200000
     let [delim, policy, comment_prefix] = rainbow_csv#get_current_dialect()
+    if (&ft == 'markdown' || &ft == 'rmd')
```

Comment:
Actually, this might not be a big deal, I can fix it myself, later, so you can leave it as is if you want. "monocolumn" is just a fall back which means that no better dialect was found.

Reply:
after reading through the rest of the file, wondering if &ft is correct choice over &syntax. Input?